include/WCSimWCHit.hh

```diff
@@ -31,7 +31,7 @@ class WCSimWCHit : public G4VHit
  public:
   
   void SetTubeID       (G4int tube)                 { tubeID = tube; };
-  void SetTrackID      (G4int track)                { trackID = track; };
+  void SetTrackID      (G4int track)                { trackID.push_back(track); };
```

Comment:
Please rename to something like `AddTrackID()` to make it clear that it's pushing back, rather than setting the only value